codemod-expand-transforms.js

```diff
@@ -0,0 +1,45 @@
+const glob = require('fast-glob')
+const fs = require('fs')
+
+const composeNewLines = line => {
+  const name = line.split('(')[0]
+  const values = line.match(/\(([^)]+)\)/)[1].split(/,\s|,/)
+  const axes = ['X', 'Y', 'Z']
+
+  return values
+    .map((val, j) => {
+      const lastItem = j === values.length - 1
+      return lastItem
+        ? `${name}${axes[j]} ${val}`
+        : `${name}${axes[j]} ${val}\n`
+    })
+    .join('')
```

Comment:
you could do ```values.map((val, j) => `${name}${axes[j]} ${val}`).join('\n')``` to avoid the conditional inside

Reply:
ok cool, i guess i could do the same in the morpher [here](https://github.com/viewstools/morph/blob/master/morph/svg-to-view.js#L32-L39) as well then.

Reply:
totally missed that 🙈 yup!